src/model/cpp_model.F90

```diff
@@ -53,18 +56,18 @@ end function aero_bridge_cpp_new_model
 
   !> Creates a C++ aerosol model
   !!
-  !! If the model is not a supported model in C++, a null pointer is returned
+  !! If the model is not a supported model in C, a null pointer is returned
```

Comment:
Isn't this a C++ model?

Reply:
copy/paste error =) I didn't do anything with the c/c++ model fortran code other than just get it to compile